src/main/java/org/opentripplanner/updater/alerts/GtfsRealtimeAlertsUpdater.java

```diff
@@ -95,7 +95,7 @@ protected void runPolling() {
 
             long feedTimestamp = feed.getHeader().getTimestamp();
             if (feedTimestamp <= lastTimestamp) {
-                LOG.info("Ignoring feed with an old timestamp.");
+                LOG.debug("Ignoring feed with an old timestamp from " + url);
```

Comment:
It might make sense to spit this into two separate cases, one when they are equal, and one when you actually get an older feed, which could result in a warning

Reply:
Changed it so.